lib/structures/ModalSubmitInteraction.js

```diff
@@ -0,0 +1,400 @@
+"use strict";
+
+const Interaction = require("./Interaction");
+const Member = require("./Member");
+const {InteractionResponseTypes} = require("../Constants");
+
+/**
+* Represents a message component interaction. See Interaction for more properties.
+* @extends Interaction
+* @prop {PrivateChannel | TextChannel | NewsChannel} channel The channel the interaction was created in. Can be partial with only the id if the channel is not cached.
```

Comment:
is this uncached documentation wording consistent with the same prop in message? it definitely doesn't match event docs